plugin/builtin/replicacounttransformer/ReplicaCountTransformer.go

```diff
@@ -35,15 +35,36 @@ func (p *plugin) Config(
 }
 
 func (p *plugin) Transform(m resmap.ResMap) error {
+
+	found := false
 	for i, replicaSpec := range p.FieldSpecs {
 		for _, res := range m.GetMatchingResourcesByOriginalId(p.createMatcher(i)) {
+			found = true
 			err := transformers.MutateField(
 				res.Map(), replicaSpec.PathSlice(),
 				replicaSpec.CreateIfNotPresent, p.addReplicas)
 			if err != nil {
 				return err
 			}
 		}
+		for _, res := range m.GetMatchingResourcesByCurrentId(p.createMatcher(i)) {
+			found = true
```

Comment:
This `for loop` is almost identical with the first one. Can you combine them two?

Reply:
Great idea!